views/js/qtiCreator/renderers/Object.js

```diff
@@ -26,7 +26,10 @@ define([
     var CreatorObject = _.clone(Renderer);
 
     CreatorObject.render = function(object, options){
-
+        if (object.attributes.data && this.getOption('transcriptionMetadata') && this.getOption('resourceMetadataUrl')) {
+            object.metaData['metadataUri'] = this.getOption('transcriptionMetadata');
```

Comment:
Differemt index names, its that ok? It doesn't looks like ok so maybe we need to align indexes to be clear?